vault/logical_system_test.go

```diff
@@ -68,6 +68,8 @@ func TestSystemBackend_RootPaths(t *testing.T) {
 		"storage/raft/snapshot-auto/config/*",
 		"leases",
 		"internal/inspect/*",
+		"seal",
+		"step-down",
 	}
```

Comment:
Note to reviewer: I wonder if we might instead just **delete** this test entirely? It is just retrieving the value of the root paths slice, which is defined as a literal in the main code, and then comparing it to ... another identical literal definition.

It seems to me this is a pointless thing to be testing - of course two literals that are identically defined, will have equal values.

Reply:
I also don't see much value in this particular test and wouldn't object if it was deleted.

Reply:
Updated to removed the test.